dts/common/nordic/nrf54h20.dtsi

```diff
@@ -168,6 +168,20 @@
 				interrupts = <64 NRF_DEFAULT_IRQ_PRIORITY>,
 					     <65 NRF_DEFAULT_IRQ_PRIORITY>;
 			};
+
+			wdt010: watchdog@14000 {
```

Comment:
Please use unique labels in this file, such as:
```suggestion
			cpuapp_wdt010: watchdog@14000 {
```
Then, you can add localized labels to the following files in `dts/arm/nordic`:

```
/* nrf54h20_cpuapp.dtsi */

wdt010: &cpuapp_wdt010 {};
wdt011: &cpuapp_wdt011 {};

/* nrf54h20_cpurad.dtsi */

wdt010: &cpurad_wdt010 {};
wdt011: &cpurad_wdt011 {};
```

Reply:
I applied changes. Check if this is what You meant.